src/redux/store.ts

```diff
@@ -0,0 +1,7 @@
+import bookmarkNewsReducer from "./reducers/bookmarkNewsReducer";
+import { createStore } from "redux";
+
+
+const store = createStore(bookmarkNewsReducer)
```

Comment:
currently if you initialize the store in this way, you won't be able to debug your state using redux devtools. Try to initialize it in a way so that you can debug using devtools. Add Redux extension to chrome if you haven't yet. https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en

Reply:
Fixed